### PR TITLE
adding allen & miller data

### PR DESCRIPTION
**Pull request by joselyn-rodriguez**

### CODE REVIEW

**Comment by kleinschmidt** [comment]
Can you also add the citation/description to the README, R/data.R, and the CITATION file?

File: R/data.R [hunk 1]
```diff
@@ -34,6 +34,11 @@ NULL
 #'   Corpus of conversational speech by Andy Wedel. Age is coded as old (40+) or
 #'   young (under 40).  Includes two measures of local speech rate, syllables
 #'   per second and total stop length.}
+#'   
+#' \item{allen-miller}{Allen & Miller (1999). Experiment 1 was similar to \code{gva13}
+#'   and measured VOTs for voiced stops in monosyllabic words. Experiment 2 was the
```
Comment (kleinschmidt) [inline R/data.R]: voiced and voiceless right?

**Comment by kleinschmidt** [comment]
A few more things: don't include the .DS_store file (that's a macOS thing) and re-run `devtools::document()` to update the stored document file (and commit that as well).

File: R/data.R (hunk 1 above)
Comment (joselyn-rodriguez) [reply R/data.R]: oh right - I thought I had corrected that. Ill fix that now

**Pull request merged**